### PR TITLE
Add content contribution docs

### PR DESCRIPTION
**Pull request by patricksimonian**
## Summary
Updated/added docs in prep to surface them in the devhub for content contributors

## Notable Changes 
- updated registry.md
- added a small sla
- added a github issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/content_suggestion.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+__Where does the content live (Github or a Website)?__
+
+*replace with answer*
+
+__What is the url to the content (if github, the url to the file on github)?__
+
+*replace with answer*
+
+__What Topic or Journeys should this content belong too (comma separated)?__
```
Comment (ShellyXueHan) [inline .github/ISSUE_TEMPLATE/content_suggestion.md]: typo: too -> to

File: docs/registry.md [hunk 2]
```diff
@@ -9,31 +8,71 @@ tags:
...
+  - new topic
+  - new journey
+  - edit content
 ---
+# Contributing To The Devhub
+
+The Devhub's content contribution model allows it to source content from a variety of sources and stitch them together. 
+The way it achieves this is through a set of loosely coupled files called __registries__. 
+
+The two ways you two get content into the Devhub is by either _making a pull request_ or _[creating a new github issue](https://github.com/bcgov/devhub-app-web/issues/new)_.
```
Comment (ShellyXueHan) [inline docs/registry.md]: `The two ways you two get`

File: .github/ISSUE_TEMPLATE/content_suggestion.md (hunk 1 above)
Comment (ShellyXueHan) [reply .github/ISSUE_TEMPLATE/content_suggestion.md]: where can one find the available topics/journeys?

Comment (patricksimonian) [reply .github/ISSUE_TEMPLATE/content_suggestion.md]: graciass

**Pull request merged**